File: 2020/02/a.cpp
#include <algorithm>
#include <iostream>

using namespace std;

int
main(int, char *[])
{
    ios::sync_with_stdio(false);
    unsigned valid = 0, lo, hi;
    char sep, ch;
    string pw;
    while (getline(cin >> lo >> sep >> hi >> ws >> ch >> sep >> ws, pw)) {
        auto n = count(pw.begin(), pw.end(), ch);
        if (lo <= n && n <= hi)
            ++valid;
    }
    cout << valid << endl;
    return 0;
}

File: 2020/02/a.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	var valid int
	for {
		var lo, hi int
		var ch, pw string
		_, err := fmt.Scanf("%v-%v %1v: %v\n", &lo, &hi, &ch, &pw)
		if err != nil {
			break
		}
		n := strings.Count(pw, ch)
		if lo <= n && n <= hi {
			valid++
		}
	}
	fmt.Println(valid)
}
